packages/datagateway-search/src/searchPageCardView.component.test.tsx

```diff
@@ -67,6 +70,8 @@ describe('SearchPageCardView', () => {
     props = {
       onTabChange: onTabChange,
       currentTab: 'investigation',
+      cartItems: useCart,
```

Comment:
technically this should be an array right? I know it probably doesn't affect the test but the type here is wrong